### DISCUSSION TITLE
Application hanging (not showing QR code) at start time ?

### DISCUSSION THREAD

**Discussion by manuelselvaf** [eneshecan/whatsapp-for-linux#417] [Q&A] [answered]
I installed (I tried the 2 last releases and built from sources) WasIstLos easily. The application starts without troubles, but on the first screen showing normally the QR code the app is hanging : the QR code does not show up and clicking on "link with phone number" has not effect.

I am running Debian 12 with xmonad window manager. Does anyone have a clue on what may be happening ?

Thank you very much

**Comment by DMel74** [comment]
I get the same installing the flatpak on LinuxMint 22

Kernel: 6.8.0-41-generic arch: x86_64 bits: 64 compiler: gcc v: 13.2.0 clocksource: hpet
  Desktop: Cinnamon v: 6.2.9 tk: GTK v: 3.24.41 wm: Muffin v: 6.2.0 vt: 7 dm: LightDM v: 1.30.0
    Distro: Linux Mint 22 Wilma base: Ubuntu 24.04 noble

**Reply by manuelselvaf** [reply]
Sorry, but I am happy not to be alone :) And more seriously, I would be VERY happy to be able to use WasIstLos for the following reasons :
- tray icon integration
- zoom in and out shortcuts

**Reply by manuelselvaf** [reply]
Hi there, any news on this issue ?

Thank you very much

**Comment by DMel74** [comment, marked as answer]
I have heard nothing.

I ended up uninstalling, rebooting, and reinstalling (2x) and it finally started working again.

Sent with [Proton Mail](https://proton.me/mail/home) secure email.

**Reply by manuelselvaf** [reply]
Thank you very much. I did the same, and now it is ok.

**Comment by manuelselvaf** [comment]
Uninstall and reinstall solve the issue.
